src/market_mad/remotes/http/import

```diff
@@ -77,7 +77,7 @@ if [ -n "$BRIDGE_LIST" ]; then
     DST_HOST=`get_destination_host $ID`
     CP_CMD="$UTILS_PATH/downloader.sh -n '${IMPORT_SOURCE}' -"
 
-    multiline_exec_and_log "set -e -o pipefail; $CP_CMD | $SSH ${DST_HOST} $DD of=${DST_PATH} bs=64k" \
+    multiline_exec_and_log "set -e -o pipefail; $CP_CMD | $SSH ${DST_HOST} $DD of=${DST_PATH} bs=${DD_BLOCK_SIZE:-64k}" \
```

Comment:
Where could this DD_BLOCK_SIZE set? Is there any conf file for market?

Reply:
> Where could this DD_BLOCK_SIZE set
for now the only way to set is via global environment variables

> Is there any conf file for market?
Still not, but I can prepare it.